source/making_payments/index.html.md.erb

```diff
@@ -0,0 +1,124 @@
+---
+title: Making payments 
+weight: 70
+---
+
+# Making payments
+
+When you make a payment, you will need to supply a ``reference``. This is a
+unique identifier for the payment. If your organisation already has an
+existing identifier for payments, you can use it here.
+
+You will also need to supply a ``return_url``, a URL hosted by your service
+for the user to return to after they have completed payment on GOV.UK Pay. See
+the section below on **Choosing the return URL** for more information.
+
+You will need to store the URL from the Location header/in the ``self``
+section of ``links`` in the JSON body (the same URL is shown in both places).
+This URL contains the GOV.UK Pay ``payment_id`` which uniquely identifies the
+payment. An authenticated GET request to the URL will return information about
+the payment and its status.
```

Comment:
It feels like this might be a good place to refer the reading to the reporting section